capstone-app/src/app/modules/photos/components/photos-section/photos-section.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-photos-section',
+  templateUrl: './photos-section.component.html',
+  styleUrls: ['./photos-section.component.scss']
+})
+export class PhotosSectionComponent implements OnInit {
```

Comment:
What's the purpose of this wrapper component? Why not just use image container for everything (renaming if necessary).

Reply:
This component represents the entire photos section which will eventually consist of the images themselves, a title, filtering options, setting results limits, and any other future features. The image container just consists of the flex box of images and their hover features